jobs/performance_tests/templates/run_perf_test.erb

```diff
@@ -67,29 +90,9 @@ run_perf_tests() {
   NUM_OK=$(grep -E "\[200\].*responses" ${LOG_DIR}/routed_loadtest_${date}.log | awk '{ print $2 }')
 }
 
-emit_datadog_metrics() {
```

Comment:
Do we really want to delete this function? And if we do, then we should probably remove the call to it on line 104 of this same file.

Reply:
We don't use datadog anymore that I know of. This is safe to delete.

Reply:
cool, we'll delete *all* the datadog stuff from the release when we clean it up

Reply:
👍